tests/testthat/test-derive_summary_records.R

```diff
@@ -84,7 +84,7 @@ test_that("derive_summary_records Test 3: Errors", {
   # Is by_vars `exprs()` object?
   expect_error(
```

Comment:
Could you use `class = "assert_data_frame"` in both `expect_error()` calls. Then the first call doesn't accept _any_ error and the second call doesn't fail if the wording of the error message changes.